tests/Database/stubs/PDOExceptionStub.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+class PDOExceptionStub extends PDOException
```

Comment:
Instead of a separate file and adding it manually to Composer's autoload list, just define the class below the DatabaseConnectionTest class in the same file.

Reply:
Fixed in 3c3daad.